packages/suir-component-mapper/src/files/form-template.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button as SuirButton, Header } from 'semantic-ui-react';
+
+import FormTemplate from '@data-driven-forms/common/src/form-template';
+
+const Form = ({ children, ...props }) => (
+  <form className="ui form" noValidate {...props}>
```

Comment:
is this semantic ui specific classname?

Reply:
Yep. The form component does not allow the noValidate prop form some reason